dubbo-config/dubbo-config-spring/src/main/java/org/apache/dubbo/config/spring/context/annotation/DubboConfigConfigurationRegistrar.java

```diff
@@ -17,44 +17,69 @@
 package org.apache.dubbo.config.spring.context.annotation;
 
 import org.apache.dubbo.config.AbstractConfig;
-
-import org.springframework.context.annotation.ImportSelector;
+import org.springframework.beans.factory.support.BeanDefinitionRegistry;
+import org.springframework.context.EnvironmentAware;
+import org.springframework.context.annotation.ImportBeanDefinitionRegistrar;
 import org.springframework.core.Ordered;
 import org.springframework.core.annotation.AnnotationAttributes;
+import org.springframework.core.env.Environment;
 import org.springframework.core.type.AnnotationMetadata;
 
+import static org.apache.dubbo.config.spring.util.AnnotatedBeanDefinitionRegistryUtils.registerBeans;
+
 /**
- * Dubbo {@link AbstractConfig Config} Registrar
+ * Dubbo {@link AbstractConfig Config} {@link ImportBeanDefinitionRegistrar register}, which order can be configured
  *
  * @see EnableDubboConfig
  * @see DubboConfigConfiguration
+ * @see Ordered
  * @since 2.5.8
  */
-public class DubboConfigConfigurationSelector implements ImportSelector, Ordered {
+public class DubboConfigConfigurationRegistrar implements ImportBeanDefinitionRegistrar, EnvironmentAware, Ordered {
+
+    /**
+     * The property name of {@link EnableDubboConfig}'s {@link Ordered order}
+     */
+    public static final String DUBBO_CONFIG_ORDER_PROPERTY_NAME = "dubbo.config.order";
+
+    private int order;
```

Comment:
questions:
1. I am not sure If we need setOrder to set.
2. is it need a default value?

Reply:
I have seen the default value in setEnvironment method.